codexctl/device.py

```diff
@@ -15,6 +16,62 @@
     pass
 
 
+class HardwareType(enum.Enum):
+    RM1 = enum.auto()
+    RM2 = enum.auto()
+    RMPP = enum.auto()
+
+    @classmethod
+    def parse(cls, device_type: str) -> "HardwareType":
+        if device_type.lower() in ("pp", "pro", "rmpp", "ferrari", "remarkable ferrari"):
+            return cls.RMPP
+        elif device_type.lower() in ("2", "rm2", "remarkable 2", "tests"):

```

Comment:
```suggestion
        elif device_type.lower() in ("2", "rm2", "remarkable 2"):

```

Reply:
There are no local tests outside of `test.py` or basic sanity checks that don't look to replicate running on device. @Jayy001 may have done some local testing by forcing some things, but that would not be good to leave enabled in the main application, and is better handled by either hiding it behind an environment variable to turn on, or build into `test.py` to trigger the explicit logic.

Reply:
Agreed, I've handled this in *test.py* by using a mock object.